src/libs/Notification/PushNotification/index.native.js

```diff
@@ -30,21 +31,21 @@ function pushNotificationEventCallback(eventType, notification) {
         payload = JSON.parse(payload);
     }
 
-    Log.info(`[PUSH_NOTIFICATION] Callback triggered for ${eventType}`);
+    Log.info(`[PushNotification] Callback triggered for ${eventType}`);
```

Comment:
ATM we don't have any pre-defined log searches pointing to these kind of logs. This is more so for local debugging than anything so we should be safe to change this. I changed the tags for these since it's inconsistent with the rest of the PascalCased logs in our app